src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/ColorRange.cs

```diff
@@ -55,7 +55,7 @@ public void CustomizeView(ColorRange model, NodeView nodeView)
                         var data = colorsMirror.GetData();
                         if (data.IsCollection)
                         {
-                            colors.AddRange(data.GetElements().Select(e => e.Data).Cast<Color>());
+                            colors.AddRange(data.GetElements().Select(e => e.Data).OfType<Color>());
```

Comment:
I assume OfType should be just be filtering all Color types.
If so then looks good